community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -151,6 +149,27 @@ public void shouldImportWithHeadersBeingInSeparateFiles() throws Exception
     }
 
     @Test
+    public void shouldPrintWarningsIfHeaderHasLessColumnsThanData() throws Exception
+    {
+        // GIVEN
+        List<String> nodeIds = nodeIds();
+        Configuration config = Configuration.TABS;
+
+        // WHEN data file contains more columns than header file
+        int extraColumns = 3;
+        String output = executeImportAndCatchOutput( "--into", dbRule.getStoreDirAbsolutePath(), "--delimiter", "TAB",
+                "--array-delimiter", String.valueOf( config.arrayDelimiter() ), "--nodes", nodeHeader( config )
+                        .getAbsolutePath() + MULTI_FILE_DELIMITER + nodeData( false, config, nodeIds, alwaysTrue(),
+                        Charset.defaultCharset(), extraColumns ).getAbsolutePath(), "--relationships",
+                relationshipHeader( config ).getAbsolutePath() + MULTI_FILE_DELIMITER + relationshipData( false, config,
+                        nodeIds, alwaysTrue(), true ).getAbsolutePath() );
```

Comment:
This is too dense to read. Let's put this one option per line.

Something like:
```
executeImportAndCatchOutput(
        "--into", dbRule.getStoreDirAbsolutePath(),
        "--delimiter", "TAB",
        "--array-delimiter", String.valueOf( config.arrayDelimiter() ),
        "--nodes", nodeHeader( config ).getAbsolutePath() + MULTI_FILE_DELIMITER
                + nodeData( false, config, nodeIds, alwaysTrue(), Charset.defaultCharset(), extraColumns ).getAbsolutePath(),
        "--relationships", relationshipHeader( config ).getAbsolutePath() + MULTI_FILE_DELIMITER
                + relationshipData( false, config, nodeIds, alwaysTrue(), true ).getAbsolutePath() );
```

(but perhaps more line breaks are needed)